src/containers/StudyEditForm/StudyEditForm.tsx

```diff
@@ -0,0 +1,349 @@
+import * as Yup from 'yup';
+import { useNavigate, useParams } from 'react-router';
+import { useDispatch } from 'react-redux';
+import React, { useEffect, useState } from 'react';
+import { Formik, Field } from 'formik';
+import { AxiosError, AxiosResponse } from 'axios';
+import { openAlert } from '@store/slices/flashAlert';
+import { errorType } from '@src/interfaces/error';
+import {
+  MultiLineInput,
+  LabelInput,
+  FileInput,
+} from '@src/components/StudyCreate&Edit';
+import { editStudy } from '@src/apis/studyEdit';
+import { getStudyDetailInfomation } from '@src/apis/studyDetail';
```

Comment:
p3;
이것들도 위와 같습니다. ;) 

```suggestion
} from '@components/StudyCreate&Edit';
import { editStudy } from '@apis/studyEdit';
import { getStudyDetailInfomation } from '@apis/studyDetail';
```

Reply:
이것도 반영 완료! b78f2b6547903fef6e67810e92cde4877af8f254